### ISSUE TITLE
Bug: the app doesn't auto starts with windows, the option to start with windows is on still it never automatically starts with windows and i have to launch it manually every time.

### ISSUE THREAD

**Issue by EmpireAbhi** [ModernFlyouts-Community/ModernFlyouts#437]
**To Reproduce:**

**Screenshots:**

**OS Version:**
 - Windows 10 Version: 

**ModernFlyouts Version:**
 - Version: 

**Additional context:**

**Comment by ADeltaX** [comment]
Does EventViewer > Application say something about the program (e.g. it crashed right after starting windows)?

Also, what is your OS version?

**Comment by Samuel12321** [comment]
also what Anti virus are you using? @EmpireAbhi

**Comment by EmpireAbhi** [comment]
> also what Anti virus are you using? @EmpireAbhi

just Microsoft security/defender

**Comment by EmpireAbhi** [comment]
> Does EventViewer > Application say something about the program (e.g. it crashed right after starting windows)?
> 
> Also, what is your OS version?

windows 10 home version: 1909
os build: 18363.1379

**Comment by EmpireAbhi** [comment]
> Does EventViewer > Application say something about the program (e.g. it crashed right after starting windows)?
> 
> Also, what is your OS version?

no, no messages or anything of that kind. no dialogue box, no notification, no crash, nothing.

**Comment by Cyberdroid1** [comment]
@EmpireAbhi 
Check the Reliability monitor and see if there are any crashes related to ModernFlyouts.

**Comment by fillwithjoy1** [comment]
There is a method however
Use **Ctrl+R** to open Run and type "Shell:AppsFolder"
Scroll down to ModernFlyouts (Preview) and right click it > Create Shortcut
It'll say that Windows cannot place the shortcut here and can only place it in the desktop. Press Yes
Now **Ctrl+R** again and type "Shell:Startup"
You have the startup folder and drag the shortcut from the desktop to the folder.

This is a alternative method to force windows to automatically start an app on startup

**Comment by Samuel12321** [comment]
@EmpireAbhi is this resolved by 0.9.1?

**Comment by iamshilov** [comment]
I have the same issue. But I'd say even more. Not only does it not autostart with the option enabled, the app also unloads itself (or is being killed) after some time.

**Comment by mizzunet** [comment]
same here, ModerFlyouts doesn't seem to be opened at the starting, because first usage of volume/beightness up/down is the normal UI not ModernFlyout. I'm ready to give any kind of logs if you need

**Comment by fillwithjoy1** [comment]
Now it's happened to me, it doesn't activate until i actually open the app
and after a certain period of time the modernflyout doesn't appear, nor the
legacy flyout either until I open the app

**Comment by Samuel12321** [comment]
@fillwithjoy1 @mizzunet @iamshilov 
Is this still occurring with the current release (0.9.3)?

**Comment by iamshilov** [comment]
@Samuel12321 yep

**Comment by fillwithjoy1** [comment]
Yep

**Comment by Cyberdroid1** [comment]
Well, I just faced this issue myself yesterday.

Launching it normally from Start works just fine.

**Comment by fillwithjoy1** [comment]
Same here

**Comment by JohnLGalt** [comment]
I'm on Windows 11, build 22000.120, which was delivered and installed yesterday, and this happened to me for the first time with ModernFlyouts.

I didn't notice it until I started playing music, and then saw it was not running.

But I did a reboot last night before going to bed, and when logging in today it was already running.  So it seems to be random on my end.

**Comment by fillwithjoy1** [comment]
It always happens after some hours or some time

**Comment by fillwithjoy1** [comment]
Possible UWP Optimization Issues?

**Comment by Cyberdroid1** [comment]
Modernflyouts is not a UWP app, its a packaged Win32 app.

**Comment by fillwithjoy1** [comment]
Happens to me some of the time (quoting Sammy Guergachi)

**Comment by koopa990** [comment]
> There is a method however Use **Ctrl+R** to open Run and type "Shell:AppsFolder" Scroll down to ModernFlyouts (Preview) and right click it > Create Shortcut It'll say that Windows cannot place the shortcut here and can only place it in the desktop. Press Yes Now **Ctrl+R** again and type "Shell:Startup" You have the startup folder and drag the shortcut from the desktop to the folder.
> 
> _This is a alternative method to force windows to automatically start an app on startup_ _It is worth noting that it'll not appear in startup apps in settings, so if you need to delete it, you can manually do this_ _**CTRL+R** and "Shell:Startup", then delete the shortcut_

This works for me!
